spec/index.html

```diff
@@ -6370,9 +6275,8 @@ <h5>DATATYPE</h5>
             <p>Returns the <span class="type datatypeIRI">datatype IRI</span> of a
               <code>literal</code>.</p>
             <ul>
-              <li>If the literal is a typed literal, return the datatype IRI.</li>
-              <li>If the literal is a simple literal, return <code>xsd:string</code></li>
-              <li>If the literal is literal with a language tag, return
+              <li>If the literal is a literal without language tag, return the datatype IRI.</li>
```

Comment:
```suggestion
              <li>If the literal is a literal without language tag, return the datatype IRI of this literal.</li>
```

Reply:
> Can we just remove the whole <ul> now that all literals have datatypes?

That should be sufficient indeed. Will look into removing the two cases.

I do think it make sense to add a note, reminding the reader that language-tagged strings have a dedicated datatype.